less/navigation.less

```diff
@@ -1 +1,36 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+nav {
+    width: 100%;
+    background-color: seashell;
+    z-index: 1;
+    position: fixed;
+
+    .nav {
+        width: 800px;
```

Comment:
no need to declare a px width here because that was defined in global.less...Just 100% will do.